Mixpanel/MPABTestDesignerConnection.m

```diff
@@ -15,11 +15,14 @@
 #import "MPDesignerSessionCollection.h"
 #import "MPLogger.h"
 #import "MPSwizzler.h"
+#import "MPResources.h"
+
+#define CONNECTIVITY_INDICATOR_SIZE 50
```

Comment:
Always prefer `static const` to `#define`, but can't we just do `image.size.width` rather than hard coding a magic number?